dbManager.py

```diff
@@ -1,11 +1,13 @@
 from flask import Flask
 import mysql.connector
 from _init_ import app
+import csv
+from werkzeug.security import generate_password_hash
 
 #open database connection
 def open_connection():
 	#change any of the names below depending on db creation
-	connection = mysql.connector.connect(user='admin', password='ElectionProtectionPass', host='127.0.0.1', database='electionData')
```

Comment:
I was having issues even with my admin having that password so i removed that parameter